packages/bp/src/core/messaging/messaging-service.ts

```diff
@@ -115,6 +115,21 @@ export class MessagingService {
       ? `${process.EXTERNAL_URL}/api/v1/chat/receive`
       : // We set a dummy webhook to get back a webhook token. The actual url that will be called is SPINNED_URL
         'http://dummy.com'
+
+    // Fill env variables into bot messages.channels using template:
+    // %MY_ENV_VAR%
+    if (messaging.channels) {
+      messaging.channels = Object.keys(messaging.channels).reduce((newChannels, chKey) => {
+        const { channels } = messaging as any // @ts-hack, channels will exist because of conditional above
+        newChannels[chKey] = Object.keys(channels[chKey]).reduce((channel, key) => {
+          const value: string = channels[chKey][key]
+          channel[key] = value.match(/^%.*%$/) ? process.env[value.replace(/%/g, '')] || value : value
+          return channel
+        }, {})
+        return newChannels
+      }, {})
+    }
```

Comment:
If we want to implement some kind of env var to config replacement, I suggest we use the same logic that we have for module configuration. 
![image](https://user-images.githubusercontent.com/42552874/154205711-5ce57831-2fe7-4d49-adca-49f4ae6109cd.png)

Reply:
We had this discussion like 3 months ago and you agreed on this.
Setting bot config through en var pattern becomes hard to use and needs complex documentation. Remember ?